README.md

```diff
@@ -264,6 +264,7 @@ Table of Contents
   * [rollbar.com](https://rollbar.com/) — Exception and error monitoring, free plan with 5,000 errors/month, unlimited users, 30 days retention
   * [bugsnag.com](https://bugsnag.com/) — Free for up to 2,000 errors/month after the initial trial
   * [sentry.io](https://sentry.io/) — Sentry tracks app exceptions in real-time, has a small free plan. Free, unrestricted use if self-hosted
+  * [fabric.io](https://fabric.io/kits/android/crashlytics) - Crash and error reporting for Android and iOS
```

Comment:
Traditionally I've been avoiding adding pure mobile dev things here as the list is slanted towards services conceivably of interest to infrastructure developers rather than just rando development.

So crashlytics keeps coming up here and I usually reject it, sorry but thank you for the effort

Reply:
No worries, although just curious if there has been thought about making a **Mobile** section so that Mobile focused SDKs and platforms can have a place on this list for those who need them.